p2p/utils.py

```diff
@@ -66,7 +66,7 @@ def get_block_numbers_for_request(
     return sequence_builder(block_number, limit, skip, reverse)
 
 
-def get_process_pool_executor() -> ProcessPoolExecutor:
+def get_asyncio_executor() -> Executor:
```

Comment:
@gsalgado what's the reason for this name change? Isn't `get_process_pool_executor` quite accurate?

Reply:
This function returns the executor we use in conjunction with `loop.run_in_executor()`, and an `Executor` is what we want there -- the fact that we use a `ProcessPoolExecutor` by default is just an implementation detail. We might change this to return a `ThreadPoolExecutor`, for instance, if we're running on a single-core machine with limited resources

Reply:
>the fact that we use a `ProcessPoolExecutor` by default is just an implementation detail.

Totally, but imagine we also had a `ThreadPoolExecutor`, then I guess having the functions named `get_process_pool_executor` and `get_thread_pool_executor` would make even more sense to differentiate. Now that we only have one I agree that `get_asyncio_executor` isn't wrong, I just think that the previous name was a better fit **especially** if we consider that there **could** be different functions, returning different kind of implementations.

Reply:
We won't have two different executors/functions for the `ChainSyncer` to choose from -- we'd still have a single function that returns the executor, depending on user config, machine resources, whatever. I can't help but wonder if such a trivial change really warrants that much bikeshedding, though...

Reply:
> can't help but wonder if such a trivial change really warrants that much bikeshedding

Oh, I didn't mean to do that. I was just interested to hear the reason for the rename and I continue to believe the previous name was well suited. However, naming things is highly subjective anyway so no need to continue bikeshedding here :wink: